bin/node/executor/Cargo.toml

```diff
@@ -41,7 +41,7 @@ sp-application-crypto = { version = "2.0.0-rc6", path = "../../../primitives/app
 sp-runtime = { version = "2.0.0-rc6", path = "../../../primitives/runtime" }
 sp-externalities = { version = "0.8.0-rc6", path = "../../../primitives/externalities" }
 substrate-test-client = { version = "2.0.0-rc6", path = "../../../test-utils/client" }
-wabt = "0.9.2"
+wabt = "0.10.0"
```

Comment:
We just reverted it back to `0.9.1`.

@pepyakin does `0.10.0` has the same problems?

Reply:
TBH I am not sure, but I assume there were some problems with 0.10 since the PR that contained that was closed
https://github.com/paritytech/substrate/pull/6928#event-3713122450

However, it doesn't indicate why, I guess @gnunicorn could say more.

Reply:
However, I am not sure why would this PR need to update wabt. We use wabt mostly converting WebAssembly Text Format source code into wasm binaries and thus of interest for things that do deep down wasm work.

This PR doesn't touch anything related to wasm whatsoever. 

@drewstone what is the reason for updating?

Reply:
> TBH I am not sure, but I assume there were some problems with 0.10 since the PR that contained that was closed

My understanding was that 0.10 doesn't compile unless you have latest CMAKE (or something) which some distros don't ship for some reason. @tomusdrw reported that (and an issued showed the same). As no one took it upon themselves to figure out the specifics and whether we can and should upgrade, but I was facing CI-issues with 0.9.2 being unavailable I went for the conservative approach and bumped it down to 0.9.1 for now. I have no problem with bumping it to `0.10`.

Reply:
I had 0.9.2 previously and that was throwing errors. I did not know people had downgraded yet, so I had updated it to the next version.

Reply:
I will bump it down to 0.9.1.